src/test/java/org/opensearch/neuralsearch/query/HybridQueryAggregationsIT.java

```diff
@@ -182,6 +182,57 @@ public void testAggregationNotSupportedConcurrentSearch_whenUseSamplerAgg_thenSu
         }
     }
 
+    @SneakyThrows
+    public void testPostFilter_WhenConcurrentSearchNotEnabled_thenSuccessful() {
+        updateClusterSettings("search.concurrent_segment_search.enabled", true);
+        try {
+            prepareResources(TEST_MULTI_DOC_INDEX_WITH_TEXT_AND_INT_MULTIPLE_SHARDS, SEARCH_PIPELINE);
+
+            AggregationBuilder aggsBuilder = AggregationBuilders.sum(SUM_AGGREGATION_NAME).field(INTEGER_FIELD_1);
```

Comment:
why we're adding aggs here, test supposed to test post filter?

Reply:
I also like to see more complex filter query, maybe bool with should, your current query for `range` can be one clause, also add one more, for example `match`